docs/querying/basics.md

```diff
@@ -252,6 +254,23 @@ Syntax: `<instant_query> '[' <range> ':' [<resolution>] ']' [ @ <float_literal>
 
 * `<resolution>` is optional. Default is the global evaluation interval.
 
+## Pre-processors
+
+`start()` and `end()` are the currently supported pre-processors.
+
+While they look like PromQL functions, they actually act like template variables;
+they are replaced with query start and end time respectively _before_ executing the query.
+
+For a range query, their values depend on the start and end time passed to the query API and not the evaluation time.
+
+For an instant query, `start()` and `end()` both resolve to the evaluation time.
+
+At the moment, these can only be used with `@` modifier, hence the flag `--enable-feature=promql-at-modifier` is required
```

Comment:
Can you add a link to the disabled features page.

Reply:
"… with the `@` modifier…"